src/states/running/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ target_link_libraries(running_state PUBLIC
   telemetry_task
   payload_task
   diagnostics_task
+  adcs_task
```

Comment:
let's try to keep this list sorted (I did a bad job here setting a poor precedence)